pkg/tsdb/cloudwatch/annotation_query.go

```diff
@@ -108,16 +115,16 @@ func (e *cloudWatchExecutor) executeAnnotationQuery(pluginCtx backend.PluginCont
 	return result, err
 }
 
-func transformAnnotationToTable(annotations []map[string]string, query backend.DataQuery) *data.Frame {
+func transformAnnotationToTable(annotations []*annotationEvent, query backend.DataQuery) *data.Frame {
 	frame := data.NewFrame(query.RefID,
-		data.NewField("time", nil, []string{}),
+		data.NewField("time", nil, []time.Time{}),
```

Comment:
By returning a time field instead of a string field for the annotation time, the [standard annotation parsing](https://github.com/grafana/grafana/blob/main/public/app/features/annotations/standardAnnotationSupport.ts#L47-L49) will just work automagically, so no need for transforming the response in the CW datasource file.